Gemfile

```diff
@@ -5,3 +5,4 @@ gemspec
 
 gem "rake", "~> 12.0"
 gem "rspec", "~> 3.0"
+gem "pry"
```

Comment:
Not sure what's going on here.  It's not typical for *any* gems to be listed in `Gemfile` if using `gemspec`, and I don't see any changes elsewhere in this PR that would require `pry`.

Reply:
Ah that shouldn't be there. Ta.